tensorrt_llm/_torch/custom_ops/triton_fused_inv_rope_fp8_quant.py

```
@@ -0,0 +1,312 @@
+def _fused_inv_rope_fp8_quant_fake(
+    o: torch.Tensor,
+    positions: torch.Tensor,
+    cos_sin_cache: torch.Tensor,
+    n_groups: int,
+    heads_per_group: int,
+    nope_dim: int,
+    rope_dim: int,
+    quant_group_size: int,
+    is_neox: bool,
+) -> tuple[torch.Tensor, torch.Tensor]:
+    num_tokens, num_heads, head_dim = o.shape
```

Ruff (0.15.17)

[warning] Unpacked variable `num_heads` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)